engine/interpreter/src/main/scala/pl/touk/nussknacker/engine/spel/Typer.scala

```diff
@@ -102,6 +103,16 @@ private[spel] class Typer(classLoader: ClassLoader, commonSupertypeFinder: Commo
         throw new SpelCompilationException(node, e)
     }
 
+    def indexerValidator(e: Indexer, stack: List[TypingResultWithContext]): NodeTypingResult = {
```

Comment:
maybe better name would be `typeIndexer`? like other methods in this class e.g. `typeNode`, `typeExpression`

Reply:
Done